static/js/Misc.jsx

```diff
@@ -108,21 +108,64 @@ InterfaceText.propTypes = {
   className: PropTypes.string
 };
 
-const ContentText = ({text, html, overrideLanguage, defaultToInterfaceOnBilingual=false, bilingualOrder = null}) => {
+const VersionContent = (props) => {
+  const renderedItems = ContentTextFilterByLang({...props});
```

Comment:
You don't need to expand the props here because `ContentTextFilterByLang` the same props. Instead you can simply write `ContentTextFilterByLang(props)`